test/test_analysis.py

```diff
@@ -167,3 +172,22 @@ async def test_get_analysis(self):
         ]
 
         self.assertEqual(analyses, expected)
+
+    @run_as_sync
+    async def test_get_sample_cram_path_map_for_seqr(self):
+        """
+        Exercise get_sample_cram_path_map_for_seqr()
+        """
+
+        part = await self.pl.upsert_participants(
+            [
+                ParticipantUpsertInternal(
+                    external_ids={PRIMARY_EXTERNAL_ORG: 'PEXT1'},
+                    meta={},
+                    samples=[SampleUpsertInternal(id=self.sample_id)]
+                ),
+            ],
+        )
+
+        id_map = await self.al.get_sample_cram_path_map_for_seqr(self.project_id, ['blood'], [part[0].id])
```

Comment:
Added a basic test case so that this code gets exercised. There's not enough data set up to get a non-empty `id_map` back, so we don't check the result very much.